module/metrics/transaction.go

```diff
@@ -81,11 +83,28 @@ func NewTransactionCollector(transactionTimings mempool.TransactionTimings, log
 			Subsystem: subsystemTransactionSubmission,
 			Help:      "counter for the success/failure of transaction submissions",
 		}, []string{"result"}),
+		executeScriptRTT: promauto.NewCounter(prometheus.CounterOpts{
+			Name:      "execute_script_rtt",
+			Namespace: namespaceAccess,
+			Subsystem: "TEMP",
```

Comment:
I'm not sure if I should create a dedicated Subsystem for these metrics.

Reply:
You should be able to use the existing transaction timing subsystem, I don't think these metrics warrant a new subsystem.